Mail2Bug/Email/EWS/EWSIncomingMessage.cs

```diff
@@ -10,15 +10,18 @@ namespace Mail2Bug.Email.EWS
     public class EWSIncomingMessage : IIncomingEmailMessage
     {
         private readonly EmailMessage _message;
+        private readonly byte[] _conversationId;
 
         public EWSIncomingMessage(EmailMessage message)
         {
             _message = message;
+            ExtendedPropertyDefinition conversationId = new ExtendedPropertyDefinition(0x3013, MapiPropertyType.Binary);
```

Comment:
1. Can you please add documentation of where this value is taken from (link to the exact location where this is documented)
2. Can you please extract the constant (0x3013) to an actual constant in this file